ui/src/utils/array-utils.ts

```diff
@@ -136,10 +136,6 @@ export function deduplicateById<T, TId>(items: T[], getItemId: (item: T) => TId)
     return [...x.values()];
 }
 
-export function arraysEqual<T>(arr1: T[], arr2: T[]) {
-    return JSON.stringify(arr1) === JSON.stringify(arr2);
-}
```

Comment:
Tämä oli käytännössä a) logiikaltaan täysin identtinen meidän `objectEquals`:in kanssa hieman eri tyypityksillä ja b) sitä ei muutenkaan käytetty mistään -> poistettu, käytetään `objectEquals`:ia mikäli tarpeita ilmenee